gtsam/hybrid/GaussianMixture.cpp

```diff
@@ -59,6 +59,8 @@ GaussianMixture::GaussianMixture(
                       Conditionals(discreteParents, conditionals)) {}
 
 /* *******************************************************************************/
+// TODO(dellaert): This is copy/paste: GaussianMixture should be derived from
+// GaussianMixtureFactor, no?
```

Comment:
Yes-GM should be a GMF with canonical parameters

Reply:
When you say canonical parameters, do you mean canonical parameters of the Gaussian or something else?

Reply:
> 

Yeah

Reply:
Don't worry about it, I will resolve in a subsequent PR. I think the copy/pasta is a consequence of the type erasure dance.